py/src/nupic/examples/hotgym.py

```diff
@@ -0,0 +1,161 @@
+import csv
+import datetime
+import os
+import numpy as np
+import random
+
+from nupic.bindings.algorithms import SpatialPooler
+from nupic.bindings.algorithms import TemporalMemory
+from nupic.bindings.sdr import SDR, Metrics
+from nupic.encoders.rdse import RDSE, RDSE_Parameters
+from nupic.encoders.date import DateEncoder
+from nupic.algorithms.anomaly_likelihood import AnomalyLikelihood
+
+_EXAMPLE_DIR = os.path.dirname(os.path.abspath(__file__))
+_INPUT_FILE_PATH = os.path.join(_EXAMPLE_DIR, "gymdata.csv")
```

Comment:
this is OK. Even better would be if we can place it somewhere that also the c++ example could easily reach the file